### ISSUE TITLE
Mistakes in the code

### ISSUE THREAD

**Issue by HectorAnadon** [MicrosoftDocs/windows-ai-docs#47]
In step 7 of Load the model:
hstring should be removed as the variable is already defined in step 3 of Load the model.
In step 4 of Load the image:
hstring should be removed as the variable is already defined in step 3 of Load the model.
In step 5 of Evaluate the model:
Same here. string should be removed as the variable is already defined in step 3 of Load the model.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ffd39e3a-56da-5921-4e12-23278dcd49df
* Version Independent ID: 5de3959b-c507-bb89-4a0d-448e8ff3471f
* Content: [Windows Machine Learning for Desktop (C++) tutorial](https://docs.microsoft.com/en-us/windows/ai/get-started-desktop)
* Content Source: [docs/get-started-desktop.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/get-started-desktop.md)
* Product: **windows**
* GitHub Login: @eliotcowley
* Microsoft Alias: **elcowle**

**Comment by eliotcowley** [comment]
Hi @HectorAnadon, thanks for the feedback. For your first point, you're right it is defined in step 3, but in step 7 we are rewriting the line so that it is initialized in the same step. So in this case, **hstring** should be kept.

Same deal with your second and third points&mdash;we are rewriting the variable declarations so that we initialize in the same step.

Would it help if I clarified this in the tutorial&mdash;that we are assigning the values to the variables we declared at the top?
